lib/workarea/klarna/gateway/order.rb

```diff
@@ -89,15 +89,24 @@ def format_address(address)
         def format_item(item)
           view_model = Storefront::OrderItemViewModel.new(item)
           breadcrumbs = Navigation::Breadcrumbs.from_global_id(item.via) if item.via.present?
+
           tax_amount, tax_rate = item_tax_data(item)
+          unit_price = item.current_unit_price.cents
+          total_amount = item.total_price.cents
+
+          if include_tax_in_totals?
+            tax_amount_per_item = tax_amount / item.quantity
```

Comment:
Seems like `tax_amount` would be an integer from line 129. Probably needs to be a float.

Reply:
Or maybe the intent is to round to the cents integer?